### ISSUE TITLE
Search class: Param "type" is "type_"

### ISSUE THREAD

**Issue by Dextroz** [salvoventura/pyunsplash#3]
Hey there! Nice Wrapper!

Is there a possible issue with the search class.

See following error:
`search = py_un.search(type='photos', query='red')`
`TypeError: search() missing 1 required positional argument: 'type_'`

Per your documentation/examples it says to use param "type" however, using "type_" works correctly, a possible typo?

**Comment by salvoventura** [comment]
Hello!
thanks for the notification. The documentation needs to be corrected. When using the key value, you are correct: "type_" is the correct value to use. OTOH, that is also a mandatory (positional) field, so you could get away without the key, some like:

search = py_un.search('photos', query='red')

should work.

I'll take it on me to clarify and fix the documentation on this. Thanks again!

**Comment by Dextroz** [comment]
Hey, thanks for the quick response! 
Indeed I forgot that the param name isn't required!

I have some other questions as I am using this wrapper in my own project. I have been looking and just want to clarify that the search class supports the `https://api.unsplash.com/photos/random` endpoint? by using `search = py_un.search('random', query='red')`?

Furthermore, per the Unsplash API guidelines, they ask developers to [trigger a download](https://medium.com/unsplash/unsplash-api-guidelines-triggering-a-download-c39b24e99e02). Is this a feature which is currently offered or in the future, will be offered through this wrapper?

Many thanks for your help!
Dex.

**Comment by salvoventura** [comment]
Hello,
 the Unsplash API for random photo maps to the **PyUnsplash.photos** class, with something like:
```
import pyunsplash
pu = pyunsplash.PyUnsplash(api_key='<your Unsplash API key>')
photos_page = pu.photos(type='random', per_page=15)
```
See the documentation for [PyUnsplash.photos](https://pyunsplash.readthedocs.io/en/latest/classes/class_pyunsplash.html?highlight=random#pyunsplash-photos-type-page-per-page-order-by).

As for the guidelines, the way I read them is different. They are not asking you to trigger a download. Rather, if you do want to trigger a download, they ask you to use the url location from the property `photo.links.download_location`. 

At this time, the `PyUnsplash.photo` class directly exposes `link_download` which maps to a different property, in particular `photo.links.download`. I can add the additional class property.

However, you can still work around this by something like:
```
import pyunsplash
pu = pyunsplash.PyUnsplash(api_key='<your Unsplash API key>')
photos = pu.photos(type='random', per_page=15)
for photo in photos.entries:
    print photo.links.get('download_location', None)
```

**Comment by salvoventura** [comment]
Fixed and closed as part of new beta release v1.0.0b7. Please update to this release to get the latest functionality.
